package.json

```diff
@@ -31,37 +31,37 @@
     "melon": "./bin/melon.js"
   },
   "scripts": {
-    "precompile": "rimraf out",
-    "compile": "node bin/melon.js compile",
-    "deploy": "JSON_RPC_ENDPOINT=ws://localhost:8545 node bin/melon.js deploy",
-    "devchain": "ganache-cli --gasLimit 0x7a1200 --defaultBalanceEther 10000000000000",
-    "prebuild": "rimraf lib",
     "build": "tsc -p tsconfig.json && tscpaths -p tsconfig.json -s ./src -o ./lib > /dev/null",
-    "postbuild": "yarn bundle",
     "bundle": "rollup -c",
+    "allchecks": "yarn clean && yarn build && yarn compile && yarn deploy && yarn lint && yarn test",
+    "clean": "rimraf lib && rimraf out",
+    "compile": "node bin/melon.js compile",
+    "deploy": "JSON_RPC_ENDPOINT=ws://localhost:8545 node bin/melon.js deploy",
     "dev": "tsc-watch -p tsconfig.json --onSuccess \"sh -c 'tscpaths -p tsconfig.json -s ./src -o ./lib > /dev/null'\"",
-    "test": "JSON_RPC_ENDPOINT=ws://localhost:8545 yarn jest --runInBand --forceExit",
+    "devchain": "ganache-cli --gasLimit 0x7a1200 --defaultBalanceEther 10000000000000",
     "format": "prettier --write src/**/*.ts",
-    "format:sol": "prettier --write src/**/*.sol && solium -d src/ --fix",
-    "lint:eslint": "eslint .",
-    "lint:tslint": "tsc --noEmit && tslint src/**/*.ts",
     "lint:prettier": "yarn prettier --list-different src/**/*.ts",
     "lint:solium": "solium --dir .",
+    "lint:tslint": "tsc --noEmit && tslint src/**/*.ts",
     "lint": "yarn lint:tslint && yarn lint:prettier",
-    "clean": "rimraf lib && rimraf out"
+    "postbuild": "yarn bundle",
+    "prebuild": "rimraf lib",
+    "precompile": "rimraf out",
+    "publish:alpha": "yarn check && yarn publish --tag alpha --new-version $(semver $npm_package_version -i prerelease --preid alpha)",
+    "test": "JSON_RPC_ENDPOINT=ws://localhost:8545 yarn jest --runInBand --forceExit"
   },
   "devDependencies": {
-    "0x.js": "1.0.7",
+    "@0x/types": "^1.3.0",
     "@types/jest": "^23.3.9",
     "@types/node": "^10.12.10",
+    "babel-polyfill": "^6.26.0",
```

Comment:
I don't think we need that.

Reply:
We do unfortunately: It seems like 0x project did not build their packages properly.